catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -52,6 +72,17 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Import-Package>
+                            !javax.enterprise.util,
+                            !org.xmlpull.v1,
+                            *
+                        </Import-Package>
```

Comment:
@pklinef At the time we were troubleshooting the dependencies, we were using bundle watch, and that strategy made us think `!org.xmlpull.v1` was unnecessary. A full build from the top proved otherwise.